security_groups/running_security_groups.go

```diff
@@ -169,6 +170,10 @@ var _ = SecurityGroupsDescribe("Security Groups", func() {
 		var clientAppName, securityGroupName string
 
 		BeforeEach(func() {
+			if !Config.GetIncludeContainerNetworking() {
+				Skip(skip_messages.SkipContainerNetworkingMessage)
+			}
+
```

Comment:
Can you move this conditional up the the BeforeEach for the SecurityGroupsDescribe that starts on line 161?  That would prevent pushing any apps.  The test is still pushing an app and takes 90 seconds to fail on my machine.